zgw/catalogi-api/src/main/kotlin/com/ritense/catalogiapi/CatalogiApiPlugin.kt

```diff
@@ -275,7 +329,7 @@ class CatalogiApiPlugin(
         }
     }
 
-    fun getStatustypen(zaakTypeUrl: URI): List<Statustype> {
+    fun getStatusTypen(zaakTypeUrl: URI): List<Statustype> {
```

Comment:
Please give this a name that differentiates it from the getStatustypen plugin action.